python/lsst/meas/base/forcedPhotImage.py

```diff
@@ -157,9 +171,10 @@ def runDataRef(self, dataRef, psfCache=None):
             )
         self.catalogCalculation.run(measCat)
 
-        self.writeOutput(dataRef, measCat)
+        return(measCat)
```

Comment:
In Python `return` is a keyword, not a function, so `return measCat`. That said, `run` should almost always return an `lsst.pipe.base.Struct` (for future expansion without breaking existing code) so I suggest:
`return lsst.pipe.base.Struct(measCat=measCat)`
and updating the `@return` documentation accordingly.